### PR TITLE
Release 1.1.3

### PR DESCRIPTION
**Pull request by ereteog**
> #related https://github.com/threatgrid/iroh/issues/4058
> #related https://github.com/threatgrid/ctia/pull/1065#issuecomment-778367584

Release [malware schema](https://github.com/threatgrid/ctim/pull/330) and fix CTIM version bump.

I messed this one, I pushed the release before the rebase and I did not have the proper changes. So 4 commits and 2 releases.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.1")
 (def schema-version "1.1.7")
 
-(defproject threatgrid/ctim "1.1.3-SNAPSHOT"
+(defproject threatgrid/ctim "1.1.5-SNAPSHOT"
```
Comment (msprunck) [inline project.clj:4]: Why do you bump from `1.1.3-SNAPSHOT` to `1.1.5-SNAPSHOT`? A mistake in the release process with `1.1.3`?

**Review by msprunck — APPROVED** [review]
LGTM!

File: project.clj (hunk 1 above)
Comment (ereteog) [reply project.clj:4]: exactly :s
I make first commit 1.1.3 that I deployed to clojars
then commit 1.1.4-SNAPSHOT
At this point I realized that I forgot to rebase, thus I did not have the changes that I was really supposed to release...
So I rebase made a commit 1.1.4 from which I made a release then bumped to 1.1.5-SNAPSHOT.
I hope that my brain is less twisted than my ankle :-)

**Comment by hisans** [comment]
QA Verified in 1.67 Release as well.

**Pull request merged**